e2e-tests/test/zks-apis.test.ts

```diff
@@ -152,3 +152,18 @@ describe("zks_getRawBlockTransactions", function () {
     expect(txns[0]["execute"]["calldata"]).to.equal(receipt.data);
   });
 });
+
+describe("zks_getConfirmedTokens", function () {
+  it("Should return empty list", async function () {
+    const tokens = await provider.send("zks_getConfirmedTokens", [0, 100]);
+    expect(tokens.length).to.equal(0);
```

Comment:
I think we should by default at least include the `ETH` token, as I think a non-empty response is necessary for the Block Explorer/Portal : 

```json
{
"l1Address":"0x0000000000000000000000000000000000000000",
"l2Address":"0x0000000000000000000000000000000000000000",
"name":"Ether",
"symbol":"ETH",
"decimals":18
}
```

Reply:
done